app/src/main/java/cm/aptoide/pt/home/HomePresenter.java

```diff
@@ -201,6 +208,32 @@ private void handleError(HomeBundlesModel.Error error) {
         });
   }
 
+  @VisibleForTesting public void handleEditorsChoiceClick() {
+    view.getLifecycle()
+        .filter(lifecycleEvent -> lifecycleEvent.equals(View.LifecycleEvent.CREATE))
+        .flatMap(__ -> view.appClicked()
+            .filter(click -> click.getBundle()
+                .getType()
+                .equals(EDITORS))
+            .doOnNext(click -> homeAnalytics.sendTapOnAppInteractEvent(click.getApp()
+                    .getRating(), click.getApp()
+                    .getPackageName(), click.getAppPosition(), click.getBundlePosition(),
+                click.getBundle()
+                    .getTag(), click.getBundle()
+                    .getContent()
+                    .size()))
+            .observeOn(viewScheduler)
+            .doOnNext(click -> homeNavigator.navigateWithEditorsPosition(click.getApp()
```

Comment:
Couldn’t this be an if in the handleAppClick second doOnNext? If(click.getBundle.getType.equals(EDITORS) navigateToEditors else navigateToAppView
This removes the need for a second method.
Probably the tests will fail anyway